### PR TITLE
Support for indirect stream events

### PR DESCRIPTION
**Pull request by jkodumal**

### CODE REVIEW

File: src/main/java/com/launchdarkly/client/LDUser.java [hunk 1]
```diff
@@ -254,7 +254,9 @@ public Builder email(String email) {
      * @return the builder
      */
     public Builder custom(String k, String v) {
-      custom.put(k, new JsonPrimitive(v));
+      if (key != null && v != null) {
```
Comment (jkodumal) [inline src/main/java/com/launchdarkly/client/LDUser.java]: Unrelated bug fixes

**Comment by pkaeding** [comment]
lgtm
